server/Procfile

```diff
@@ -1 +1 @@
-web: npm run migrate && npm run start
+web: npm run migrate && npm run undoAllSeed && npm run seed && npm run start
```

Comment:
Do we want to consider moving this over to `yarn` commands?

Reply:
@lauraGgit sure, we'll give it a try in next sprint.

Reply:
@lpendem please drop the story that is created here.

Reply:
@lauraGgit sure.